### ISSUE TITLE
A feature request for blocks that can execute JavaScript - code directly

### ISSUE THREAD

**Issue by kezb765** [thunkable/thunkable-issues#1185]
Code blocks are suggested that can execute JavaScript (JS) directly. Ideally, access to the components of an app should also be possible here, e.g. via the ID of a component. 

The following advantages are seen:
- Access to all functions in the core of JS with probably good performance.
- Access to all parameters and functions of components. There are some examples of components where only some specific methods / parameters are provided in the block editor in Thunkable. With JS, you could access *all* of them.
- With a JS block, one could build custom functions that would be available in the app.
- Since Thunkable is built on ReaktNative, which in turn is based on JS, no fundamental language problems should arise here.
- Until now, JS could be executed via the "WebViewer" component, but this was only possible to a limited extent and had its own disadvantages.

**Comment by tatiang** [comment]
Yes! I think it's even possible to detect swipe motions for screens (not just DVLs) using JS. Would love to see a block like this.
